modules/dreamview/frontend/src/renderer/index.js

```diff
@@ -349,7 +349,13 @@ class Renderer {
     }
 
     updateWorld(world) {
-        this.adc.update(this.coordinates, world.autoDrivingCar);
+        const adcPose = world.autoDrivingCar;
+        this.adc.update(this.coordinates, adcPose);
+        if (isNaN(adcPose.positionX) || isNaN(adcPose.positionY)) {
```

Comment:
isNaN(null) returns false. Do we want to check number using lodash's _.isNumber() instead?